docker-compose.yml

```diff
@@ -430,10 +461,14 @@ services:
       USER_UPLOAD_QUOTA: ${USER_UPLOAD_QUOTA}
 
       # Account DB connection details
-      ACCT_DB_TNS_NAME: ${ACCT_DB_TNS_NAME:?}
-      ACCT_DB_USER: ${ACCT_DB_LOGIN:?}
-      ACCT_DB_PASS: ${ACCT_DB_PASSWORD:?}
-      ACCT_DB_POOL_SIZE: ${ACCT_DB_POOL_SIZE:-5}
+#      ACCT_DB_TNS_NAME: ${ACCT_DB_TNS_NAME:?}
```

Comment:
If we make this default an empty string, does it still pick up the URL parts below?  Would be nice to be able to configure both ora and pg with these values.  Or maybe it's time to try your fix where we can use the url parts with oracle?